docs/sources/reference/commandline/cli.md

```diff
@@ -611,17 +612,20 @@ is not recursive.
     */temp*
     */*/temp*
     temp?
+    *.md
+    !LICENSE.md
 
 The first line above `*/temp*`, would ignore all files with names starting with
 `temp` from any subdirectory below the root directory. For example, a file named
 `/somedir/temporary.txt` would be ignored. The second line `*/*/temp*`, will
 ignore files starting with name `temp` from any subdirectory that is two levels
 below the root directory. For example, the file `/somedir/subdir/temporary.txt`
-would get ignored in this case. The last line in the above example `temp?`
+would get ignored in this case. The third line in the above example `temp?`
 will ignore the files that match the pattern from the root directory.
 For example, the files `tempa`, `tempb` are ignored from the root directory.
 Currently there is no support for regular expressions. Formats
-like `[^temp*]` are ignored.
+like `[^temp*]` are ignored. the last two lines means to ignore every markdown 
```

Comment:
I assume order matters, right?  So !LICENSE.md before *md would make the !LICENSE have no effect, right?